lib/membership/update.js

```diff
@@ -24,15 +24,29 @@ var Member = require('./member.js');
 var uuid = require('node-uuid');
 var util = require('util');
 
-function Update(address, incarnationNumber, status, localMember) {
-    this.address = address;
-    this.incarnationNumber = incarnationNumber;
-    this.status = status;
+/**
+ * Create a new Update
+ * @param {IMember} subject the subject of the update.
+ * @param {IMember} [source] the source of the update.
+ * @constructor
+ */
+function Update(subject, source) {
+    if (!(this instanceof Update)) {
```

Comment:
If it is used in most of the objects here we can keep it this way, it will make the code more consistent. I was just pointing out that it got added here and not used. But consistency will win in my book in the end.

👍